TFT/src/User/API/ScreenShot.h

```diff
@@ -6,6 +6,7 @@ extern "C" {
 #endif
 
 #include <stdbool.h>
+#include "variants.h"
```

Comment:
why did you include "variants.h" if there is no dependency here?

Reply:
because I restore these includes in one go. need to check each one.